src/components/ReviewsView.jsx

```diff
@@ -0,0 +1,51 @@
+import { Card, Spinner } from "react-bootstrap";
+import { useState, useEffect } from "react";
+import * as api from "../api";
+
+const ReviewsView = () => {
+  const [allReviews, setAllReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
```

Comment:
This will already be true as it's the default on line 7 - as this function is only ever being triggered upon first load, we can take it for granted that we don't need to re-set isLoading to true at any point.